arbnode/batch_poster.go

```diff
@@ -193,8 +193,8 @@ type BatchPosterConfig struct {
 }
 
 func (c *BatchPosterConfig) Validate() error {
-	if (c.LightClientAddress == "") != (c.HotShotUrl == "") {
-		return errors.New("light client address and hotshot URL must both be set together, or both left unset")
+	if len(c.HotShotUrls) == 0 {
+		return errors.New("HotShotUrls must not be empty")
```

Comment:
we should crash the app here

Reply:
Returning an error does crash the app. I was able to verify this by changing the test config to have no string and saw that the batch poster tests don't work in the CI. 

I noticed this while attempting to fix the vanilla nitro batch poster tests because they were triggering waiting for espresso validation due to having an empty string in the testing config for the hotshot urls. When I removed the string the test failed because the node couldn't be created.